src/org/labkey/targetedms/query/QCFolderDisplayColumnFactory.java

```diff
@@ -51,10 +52,15 @@ public void renderGridCellContents(RenderContext ctx, Writer out) throws IOExcep
                 StringBuilder sb = new StringBuilder();
                 qcContainers.forEach(qcContainer -> {
                     var url = qcContainer.getStartURL(user);
+                    if (!"null".equals(currentRunId))
```

Comment:
Please don't use "null" as an actual string value. It's much cleaner to use a null value itself. It'll mean you need to change how you get the value on line 39 too